USER_MANUAL.md

```diff
@@ -537,6 +588,24 @@ automatically. It will also automatically sort associative-commutative
 collections, and flatten the indentation of associative collections, when
 unparsing.
 
+### `public` and `private` attribute
+
+K allows users to declare certain pieces of syntax as either public or private.
+All syntax is public by default. Public syntax can be used from any module that
+imports that piece of syntax. A piece of syntax can be declared private with
+the `private` attribute. This means that that syntax can only be used in the
+module in which it is declared; it is not visible from modules that import
+that module.
+
+You can also change the default visibility of a module with the `private`
+attribute, when it is placed directly on a module. A module with the `private`
+attribute has all syntax `private` by default; this can be overridden on
+specific sentences with the `public` attribute.
+
+Note that the `private` module attribute also changes the default visiblity
+of imports; please refer to the appropriate section elsewhere in the manual
```

Comment:
Don't you have a link or reference instead of just _elsewhere_?
Sounds dismissive.

Reply:
I'm open to including a link if you know how...